src/main/resources/listLookup.json

```diff
@@ -630,5 +630,49 @@
         "restrictedProvenances": [],
         "defaultSensitivity": "",
         "isNonStrategic": false
+    },
+    "BIRMINGHAM_ADMINISTRATIVE_COURT_DAILY_CAUSE_LIST": {
+        "friendlyName": "Birmingham Administrative Court Daily Cause List",
+        "welshFriendlyName": "Rhestr Achosion Dyddiol Llys Gweinyddol Birmingham",
+        "shortenedFriendlyName": "Birmingham Administrative Court Daily Cause List",
+        "url": "birmingham-administrative-court-daily-cause-list",
+        "jurisdictions": ["Civil"],
```

Comment:
These lists will need updating to 'High Court' as part of the merge with the new subscriptions changes.

Reply:
updated the PR with master and add High Court as jurisdictionType.